### ISSUE TITLE
Missing files for 2020

### ISSUE THREAD

**Issue by BenjaminStuermer** [IMMM-SFA/diyepw#14]
The below WMOs are missing data for 2020 at present, which causes the 2019 files to fail in create_amy_epw_files.py because we can't append data from the subsequent year.

I think our options are to just accept the missing files, or to treat a missing year as equivalent to a year that just contains empty rows, which would mean we'd have valid data for most of the year but then a few hours of NAs the evening of December 31st.

The files currently affected are:

722520-12907-2019
722740-23160-2019
725750-24126-2019
723630-23047-2019
747185-03144-2019

@amandadsmith - What is your preference for how we resolve this?

**Comment by amandadsmith** [comment]
The source data for these files are present at https://www1.ncdc.noaa.gov/pub/data/noaa/isd-lite/2020/. It's possible they weren't there when the data was downloaded and placed in inputs, or that those files just got skipped over. Is it possible (and not a huge time sink) to build a web scraper that can go to the root of the source files at https://www1.ncdc.noaa.gov/pub/data/noaa/isd-lite/ and just grab the appropriate NOAA ISD-Lite source files for the years of interest?

If we do have a missing file for the "subsequent year" when we're creating a weather file for a given year, my preference is to treat it as equivalent to a year that just contains empty rows *as long as we're not exceeding the user's choice for `missing_consec_entries_high` or `missing_total_entries_high` by doing that*.

**Comment by BenjaminStuermer** [comment]
@amandadsmith For now I have added the missing 2020 files. I will make a task to cover the case that the file is well and truly missing as you describe, but as a separate issue that I think I would prefer to address after refactoring into functions.

It is definitely possible to grab the appropriate ISD-Lite files either en masse or on-demand. I am thinking of just downloading all of them with North American WMO IDs, stripping the middle number from their names, and putting them up on S3. That would make downloading them on demand much easier, since we would know the full file-name. That, too, is something I would love to do but prefer to wait on until after we've got a package and nice functions.
